spec/requests/api/v3/subjects/index_spec.rb

```diff
@@ -0,0 +1,847 @@
+            "subject_code" => "18",
+            "bursary_amount" => "26000",
+            "early_career_payments" => "2000",
+            "scholarship" => nil,
+            "subject_knowledge_enhancement_course_available" => true,
+          },
+        },
+        {
+          "id" => "38",
+          "type" => "subjects",
+          "attributes" => {
+            "subject_name" => "Japanese",
+            "subject_code" => "19",
+            "bursary_amount" => "26000",
+            "early_career_payments" => "2000",
+            "scholarship" => nil,
+            "subject_knowledge_enhancement_course_available" => true,
+          },
+        },
+        {
+          "id" => "39",
+          "type" => "subjects",
+          "attributes" => {
+            "subject_name" => "Mandarin",
+            "subject_code" => "20",
+            "bursary_amount" => "26000",
+            "early_career_payments" => "2000",
+            "scholarship" => nil,
+            "subject_knowledge_enhancement_course_available" => true,
+          },
+        },
+        {
+          "id" => "40",
+          "type" => "subjects",
+          "attributes" => {
+            "subject_name" => "Russian",
+            "subject_code" => "21",
+            "bursary_amount" => "26000",
+            "early_career_payments" => "2000",
+            "scholarship" => nil,
+            "subject_knowledge_enhancement_course_available" => true,
+          },
+        },
+        {
+          "id" => "41",
+          "type" => "subjects",
+          "attributes" => {
+            "subject_name" => "Spanish",
+            "subject_code" => "22",
+            "bursary_amount" => "26000",
+            "early_career_payments" => "2000",
+            "scholarship" => "28000",
+            "subject_knowledge_enhancement_course_available" => true,
+          },
+        },
+        {
+          "id" => "42",
+          "type" => "subjects",
+          "attributes" => {
+            "subject_name" => "Modern languages (other)",
+            "subject_code" => "24",
+            "bursary_amount" => "26000",
+            "early_career_payments" => "2000",
+            "scholarship" => nil,
+            "subject_knowledge_enhancement_course_available" => true,
+          },
+        },
+        {
+          "id" => "43",
+          "type" => "subjects",
+          "attributes" => {
+            "subject_name" => "Further education",
+            "subject_code" => "41",
+            "bursary_amount" => nil,
+            "early_career_payments" => nil,
+            "scholarship" => nil,
+            "subject_knowledge_enhancement_course_available" => nil,
+          },
+        },
+      ],
+      "jsonapi" => {
+      "version" => "1.0",
+    })
+  end
+
+  context "when specifying particular fields" do
+    let(:request_path) { "/api/v3/subjects?fields[subjects]=subject_name" }
+
+    it "returns the correct data" do
+      expect(json_response).to eq("data" => [
```

Comment:
Similar comment to above - though we're not expecting to change these files much